src/Components/Tabs/style.scss

```diff
@@ -0,0 +1,34 @@
+@import "./../../vars";
+
+.tabs-of-trust {
+    display: flex;
+    height: 65px;
+    justify-content: space-between;
+}
+
+.tabs-of-trust a.active {
+    position: relative;
+    border: none;
+    color: $white !important;
```

Comment:
Why did you have to use `!important` here? It should be removed.

Reply:
We've used `important` in order to overwrite the (default) gray color used by bootstrap. Without important the bootstrap color would be the one used.